integration/tests/tracer/test_tracer_debug_trace_transactions_callTracer.py

```diff
@@ -0,0 +1,501 @@
+import random
+
+import allure
+import pytest
+import json
+
+from deepdiff import DeepDiff
+from utils.helpers import wait_condition
+from integration.tests.basic.helpers.basic import AccountData
+from utils.web3client import NeonChainWeb3Client
+from utils.accounts import EthAccounts
+from utils.apiclient import JsonRPCSession
+from integration.tests.tracer.test_tracer_historical_methods import call_storage
+
+@allure.feature("Tracer API")
+@allure.story("Tracer API RPC calls debug method trace_transaction callTracer check")
+@pytest.mark.usefixtures("accounts", "web3_client", "tracer_api")
+class TestDebugTraceTransactionCallTracer:
```

Comment:
Do you have tests for cases:
1) creating contracts in a constructor and emitting events in constructor
2) delegatecall + logs 
3) different value for onlyTopCall
?

Reply:
1) case is added
2) we have call + logs and static call + logs, I am not sure we have to check all combinations. But if you think we should I'll fix it. Just let me know your decision
3) with onlyTopCall we have an issue: it works strange to me but I need to discuss it with Silviu. He is out before May 8 so I just created a bug NDEV-2959 and added a link to test case with skip mark for now. We will manage this case later with Silviu, Is it ok for you?